pipelines/LSSTCam/legacy_202409/cpLinearizer.yaml

```diff
@@ -0,0 +1,20 @@
+description: cp_pipe linearity calibration construction.
+instrument: lsst.obs.lsst.LsstCam
+imports:
+  - location: $CP_PIPE_DIR/pipelines/_ingredients/cpLinearizer.yaml
+tasks:
+  cpLinearizerSolve:
+    class: lsst.cp.pipe.LinearitySolveTask
+    config:
+      linearityType: Spline
+      usePhotodiode: true
+      splineGroupingColumn: CCOBCURR
+      maxLinearAdu: 120000
```

Comment:
Is this parameter no longer needed?  It seems to be gone in the new pipelines, and I just wanted to check that was intentional.  Apologies if I didn't follow the inheritance correctly and just missed it.  I think the rest of the configs are consistent.